code-challenges.test.js

```diff
@@ -17,28 +17,70 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-var num1 = 15
-// Expected output: "15 is divisible by three"
 
-var num2 = 0
+describe("when evenlyDivisible is called"); => {
+
+  it("returns '15 is divisble by three when called with 15'", () => {
+    //arrange
+    const argument = 15
+    //act
+    const actualDivisable = evenlyDivisible(argument);
+    //assert
+    expect(actualDivisable).toequal("15 is divisable by three")
```

Comment:
```suggestion
    expect(actualDivisable).toEqual("15 is divisable by three")
```
Not sure if `.toequal()` is getting recognized. did the tests run?